python_modules/libraries/dagster-components/dagster_components/utils.py

```diff
@@ -0,0 +1,14 @@
+import sys
+from pathlib import Path
+
+CLI_TEST_MODE_KEY = "use_test_component_lib"
+
+
+def ensure_dagster_components_tests_import() -> None:
+    from dagster_components import __file__ as dagster_components_init_py
+
+    dagster_components_package_root = (Path(dagster_components_init_py) / ".." / "..").resolve()
+    assert (
+        dagster_components_package_root / "dagster_components_tests"
+    ).exists(), "Could not find dagster_components_tests where expected"
+    sys.path.append(dagster_components_package_root.as_posix())
```

Comment:
this seems fishy

Reply:
This is exactly the same thing we do in dagster core (copy-pasted the function and changed some names) whenever we need to  to ensure that `dagster_tests` is importable inside test files. I forget the history of it.

Reply:
comment to that effect